manifests/tekton/tasks/base/cosa-build.yaml

```diff
@@ -20,16 +15,13 @@ spec:
           memory: 4Gi
       script: |
         #!/usr/bin/env bash
-
         set -euxo pipefail
 
-        echo $(params.version)
-
-        cd /srv/scos
-
+        cd /workspace/coreos
         cosa fetch
-
-        cosa build
+        cosa build ostree
+        cosa buildextend-extensions
```

Comment:
Should we add the `COSA_NO_KVM=1` here? or maybe next PR, once we narrow down the root cause for the slowness

Reply:
great point. I think we should first find out where exactly it'll be needed.